proto/vector.proto

```diff
@@ -3,11 +3,11 @@ package vector;
 
 import "event.proto";
 
-message EventRequest {
-  event.EventWrapper message = 1;
+message PushEventsRequest {
```

Comment:
What is the reason for renaming this to `PushEvents`? I'm not sure I understand the name..

Reply:
Is it 1 Vector 'pushing' events to another?

Reply:
Mostly consistency. This is used as the input type for the `PushEvents` service call. We also have a `HealthCheck` call that accepts `HealthCheckRequest`.

Additionally, given that we now push more than one event per request, renaming it from `Event` to `Events` made sense to me.